src/main/java/edu/rpi/legup/puzzle/masyu/MasyuCell.java

```diff
@@ -4,21 +4,21 @@
 
 import java.awt.*;
 
-public class MasyuCell extends GridCell<Integer> {
+public class MasyuCell extends GridCell<MasyuType> {
 
-    public MasyuCell(int value, Point location) {
+    public MasyuCell(MasyuType value, Point location) {
         super(value, location);
     }
 
     public MasyuType getType() {
```

Comment:
These changes make this method redundant, and any calls to this function should simply return `data` or access `data` directly.